twint/regex.py

```diff
@@ -0,0 +1,3 @@
+import re
+__cashtag_template = r'\$\w*'
+cashtag_pattern = re.compile(__cashtag_template)
```

Comment:
check the comment below. this file won't be required.

Reply:
Done (for those following along)